test/FormattedMessage.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import FormattedMessage from '../src/FormattedMessage';
+
+describe('<FormattedMessage /> rendering', () => {
+  const randomId = 'Random Id';
+  const values = {
+    valueToBeInjected: 'Random Value',
+  };
+
+  it('should have default value and alt props', () => {
+    expect(FormattedMessage.defaultProps.values).toBeDefined();
+    expect(FormattedMessage.defaultProps.alt).toBeDefined();
+  });
+
+  it('should render FormattedMessage', () => {
+    let wrapper = renderer.create(<FormattedMessage id={randomId} values={values} />);
```

Comment:
This will not test anything unless its wrapped with `TextProvider`

Reply:
If you look at the snapshot, its just injecting an empty string.

Reply:
Or if you want to test this "independently", you need to create a mock context of your own using the TextContext and then see if it picks up the strings from there.